lib/mwoffliner.lib.js

```diff
@@ -911,6 +927,99 @@ module.exports = {
                 }
             }
 
+            function storeDependencies( parsoidDoc, articleId, finished ) {
+                const filenameRadical = computeFilenameRadical()
+
+                // the encodeURI bellow is mandatory for languages with illegal letters for uri (like fa.wikipedia.org)
+                fetch(encodeURI(`${apiUrl}action=parse&format=json&page=${articleId}&prop=modules|jsconfigvars`), {
```

Comment:
It looks like the list of modules can be directly parsed-out from the HTML delivered by Parsoid. See at the end of the DOM for example https://en.wikipedia.org/w/api.php?action=visualeditor&format=json&paction=parse&page=Emmanuel